lib/main.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require_relative 'game'
+
+# Main program execution
```

Comment:
I would recommend to remove unnecessary comments. The names of your components and functions should be self-describing.

Reply:
Got it - Rubocop wanted me to add class descriptions but I'll change the configurations to ignore that cop

Reply:
It's usually you and your team that agree on specific formatting rules. Some linters have many default rules, but you should decide which ones are relevant for you.